src/parser.cpp

```diff
@@ -1352,11 +1558,20 @@ void Parser::ParserImpl::loadImport(ImportSourcePtr &importSource, const ModelPt
             auto issue = Issue::IssueImpl::create();
             issue->mPimpl->setDescription("Import from '" + node->attribute("href") + "' has an invalid attribute '" + attribute->name() + "'.");
             issue->mPimpl->mItem->mPimpl->setImportSource(importSource);
-            issue->mPimpl->setReferenceRule(Issue::ReferenceRule::IMPORT_ATTRIBUTE);
+            issue->mPimpl->setReferenceRule(Issue::ReferenceRule::IMPORT_ELEMENT);
             addIssue(issue);
         }
         attribute = attribute->next();
     }
+
+    if (!hrefAttributePresent) {
+        auto issue = Issue::IssueImpl::create();
+        issue->mPimpl->setDescription("Import does not specify an xlink href attribute.");
```

Comment:
maybe we need consistency in `xlink:href` vs `xlink href` in our error messages. Existing messages in the validator code use the former...in the one that I've easily found ;-)